types/Cache/Repository.php

```diff
@@ -8,14 +8,14 @@
 $cache = resolve(Repository::class);
 
 assertType('mixed', $cache->get('key'));
-assertType('int', $cache->get('cache', 27));
-assertType('int', $cache->get('cache', function (): int {
+assertType('mixed', $cache->get('cache', 27));
+assertType('mixed', $cache->get('cache', function (): int {
     return 26;
 }));
 
 assertType('mixed', $cache->pull('key'));
```

Comment:
@liamduckett wouldn't this assertion assert that the existing behavior is correct?

Reply:
Hey @crynobone. I don't think the current assertion exhibits the correct behaviour.

How can we be sure that `$cache->get('cache', 27)` is an `int`? What if the default isn't used (because the cache has a value under the requested key)?